### PR TITLE
feat(API): Document new query parameters

### PR DESCRIPTION
**Pull request by Varpusparvi**
- Document new query parameters

https://github.com/phrase/phrase/pull/11643
https://memsourcegroup.atlassian.net/browse/TSE-869

### CODE REVIEW

**Comment by Varpusparvi** [comment]
I will try to bump the GO version on the generated CLI PR @jablan

File: paths/comments/index.yaml [hunk 1]
```diff
@@ -44,12 +66,43 @@ x-code-samples:
...
+      -X GET \
+      -d '{"branch":"my-feature-branch", "filters": ["unread", "read"], "query": "Some comment content", "locale_ids": ["someId"] }' \
+      -H 'Content-Type: application/json'
 - lang: CLI v2
   source: |-
     phrase comments list \
     --project_id <project_id> \
     --key_id <key_id> \
-    --branch my-feature-branch \
+    --data '{"branch":"my-feature-branch", "filters": ["unread", "read"], "query": "Some comment content", "locale_ids": ["someId"] }' \
```
Comment (jablan) [inline paths/comments/index.yaml:78]: I believe this is a GET request, so the parameters should be passed as query params, not as JSON?

Comment (Varpusparvi) [reply paths/comments/index.yaml]: Adjusted. This is probably something we have to check for every endpoint also, seems like it's pretty mixed atm 🤔

File: paths/comment_replies/index.yaml [hunk 2]
```diff
@@ -49,14 +63,38 @@ x-code-samples:
     curl "https://api.phrase.com/v2/projects/:project_id/keys/:key_id/comments/:comment_id/replies" \
       -u USERNAME_OR_ACCESS_TOKEN \
       -X GET \
-      -d '{"branch":"my-feature-branch"}' \
+      -d '{"branch":"my-feature-branch", "filters": ["unread", "read"], "query": "Some comment content"}' \
```
Comment (jablan) [inline paths/comment_replies/index.yaml:66]: this should go into query I believe

File: paths/comments/index.yaml [hunk 3]
```diff
@@ -44,12 +66,46 @@ x-code-samples:
 - lang: Curl
   source: |-
     curl "https://api.phrase.com/v2/projects/:project_id/keys/:key_id/comments?branch=my-feature-branch" \
-      -u USERNAME_OR_ACCESS_TOKEN
+      -u USERNAME_OR_ACCESS_TOKEN \
+      -X GET \
+      -d '{"branch":"my-feature-branch", "filters": ["unread", "read"], "query": "Some comment content", "locale_ids": ["someId"] }' \
```
Comment (jablan) [inline paths/comments/index.yaml:71]: same here

**Review by jablan — CHANGES_REQUESTED** [review]
the curl examples seem wrong, with parameters in a `data` payload rather than in the query. have you tried these examples against the app, I'm curious whether both ways of passing them work?

**Review by jablan — APPROVED** [review]

**Pull request merged**